packaging/root/etc/rundeck/profile

```diff
@@ -36,13 +36,15 @@ export RDECK_JVM="-Djava.security.auth.login.config=/etc/rundeck/jaas-loginmodul
 #
 # Set min/max heap size
 #
-RDECK_JVM="$RDECK_JVM -Xmx1024m -Xms256m -XX:MaxPermSize=256m -server"
+RDECK_JVM="$RDECK_JVM -Xmx1024m -Xms256m -server"
```

Comment:
But Java 7 is still supported, right? I don't think removing the option is the best solution.

Btw, Java 8 brings a new flag:
>MaxMetaspaceSize: [...] allowing you to limit the amount of native memory used for class metadata. If you don’t specify this flag, the Metaspace will dynamically re-size depending of the application demand at runtime.

[http://javaeesupportpatterns.blogspot.ch/2013/02/java-8-from-permgen-to-metaspace.html](http://javaeesupportpatterns.blogspot.ch/2013/02/java-8-from-permgen-to-metaspace.html)

Cheers,
Daniel.